### PR TITLE
Generate table according to columns metadata

### PR DESCRIPTION
**Pull request by yonatan-roth**
## Status
- [x] Ready

## Related Issues
Fixes https://github.com/demisto/etc/issues/34523

## Related PRs
PR https://github.com/demisto/web-client/pull/7887

## Description
Print report Using table columns width from preview instead of auto-width:
- Use table columns metadata to set columns width

## Screenshots
image

### CODE REVIEW

File: src/components/Layouts/ReportLayout.less [hunk 1]
```diff
@@ -163,6 +163,10 @@ h1 {
         }
 
         @media print {
```
Comment (h88) [inline src/components/Layouts/ReportLayout.less:165]: what's the point of making a print media query here when it's the main and only use case for the UI?

Comment (yonatan-roth) [reply src/components/Layouts/ReportLayout.less:165]: @yanyo87 can you add info on that?

Comment (yanyo87) [reply src/components/Layouts/ReportLayout.less:165]: from my point of view, 
the purpose of this media query is to hide scrollbars when printing,
but to show them when developing

**Review by yanyo87 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**